src/css/doc.css

```diff
@@ -235,14 +234,12 @@ strong.beta {
   white-space: nowrap;
 }
 
-.doc a[href*="cloud.hazelcast.com"]:hover,
-.doc a[href*="viridian.hazelcast.com"]:hover {
+.doc a[href*="cloud.hazelcast.com"]:hover {
   text-decoration: none;
   color: var(--navbar-button-font-color);
 }
 
-.doc a[href*="cloud.hazelcast.com"]::after,
-.doc a[href*="viridian.hazelcast.com"]::after {
+.doc a[href*="cloud.hazelcast.com"]::after {
```

Comment:
A naive attempt to see if cloud.hazelcast.com is working was not successful. Are they gonna change the URL?

Reply:
Yes, the DNS change is currently in progress https://hazelcast.atlassian.net/browse/HZC-6537